specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/stable/2022-11-08/postgresqlhsc.json

```diff
@@ -756,7 +756,7 @@
         "tags": [
           "Configurations"
         ],
-        "operationId": "Configurations_CreateOrUpdateNode",
+        "operationId": "Configurations_UpdateOnNode",
```

Comment:
Thanks for the review, our PM team requested this change since the previous version, "Configurations - **Update Node**" could cause misunderstanding, it doesn't really update the node, only updates configuration on the node or coordinator.